controlsfx/src/main/java/org/controlsfx/control/spreadsheet/SpreadsheetCell.java

```diff
@@ -138,7 +138,29 @@
      * exceeds the width of the {@code Labeled}
      */
     public boolean isWrapText();
-    
+
+    /**
+     * Returns {@code true} if this cell contains something particular in its
+     * item and a {@code Node} given by the {@link CellGraphicFactory} will be used to
+     * display it.
+     * 
+     * @return {@code true} if this cell item needs to be given to a particular
+     * Node
+     */
+    public boolean isCellGraphic();
+
+    /**
+     * If {@code isCellGraphic} is {@code true}, this cell item contains
+     * something particular and should be display by using a Node provided by
+     * {@link CellGraphicFactory} object in the CellView.
+     *
+     * If you only seek to place a simple {@code Node} for this cell, simply use {@link #setGraphic(javafx.scene.Node)
+     * }.
+     *
+     * @param isCellGraphic if {@code true}, a Node will be used to display
+     * something particular for the cell
+     */
+    public void setCellGraphic(boolean isCellGraphic);
```

Comment:
The API is a bit confusing, especially because `setCellGraphic` is existing. I tried to be explicit in the JavaDoc to differentiate both. `setGraphic` is simply a shortcut to the official JavaFX method. `setCellGraphic` is a new method only accepting a boolean, and the `CellView` once actually showed will request the graphic to show, and return the graphic once hidden.